src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(1.2);
```

Comment:
option `transition` also must be for `color`

Reply:
done

Reply:
don`t understand at first time, but already change)